sysdig/internal/client/secure/models.go

```diff
@@ -416,3 +416,68 @@ func BenchmarkTaskFromJSON(body []byte) *BenchmarkTask {
 
 	return &result
 }
+
+// -------- Scanning Policies --------
+type ScanningPolicy struct {
+	ID             string         `json:"id,omitempty"`
+	Version        string         `json:"version,omitempty"`
+	Name           string         `json:"name"`
+	Comment        string         `json:"comment"`
+	IsDefault      bool           `json:"isDefault,omitempty"`
+	PolicyBundleId string         `json:"policyBundleId,omitempty"`
+	Rules          []ScanningGate `json:"rules"`
+}
+
+type ScanningGate struct {
+	ID      string              `json:"id,omitempty"`
+	Gate    string              `json:"gate"`
+	Trigger string              `json:"trigger"`
+	Action  string              `json:"action"`
+	Params  []ScanningGateParam `json:"params"`
+}
+
+type ScanningGateParam struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+func (policy *ScanningPolicy) ToJSON() io.Reader {
+	payload, _ := json.Marshal(policy)
```

Comment:
Ignoring error here is bad practice but I can see that this "ignored" `ToJSON` function is copy pasted everywhere in TP code. Is there any reason why is written like this?

Reply:
I see now that error is ignored in most of the places in code, why is that?

Reply:
afaik, terraform validates the JSON is "valid" in format and later we do the request where we grab the error code. (i am new to terraform development so we can add error validation there if it's necessary)